service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CloneControlledGcsBucketResourceFlight.java

```diff
@@ -84,13 +84,12 @@ public CloneControlledGcsBucketResourceFlight(
             new SetBucketRolesStep(
                 sourceBucket,
                 flightBeanBag.getGcpCloudContextService(),
-                flightBeanBag.getBucketCloneRolesComponent(),
+                flightBeanBag.getBucketCloneRolesService(),
```

Comment:
Again, this seems out of place in this PR

Reply:
I responded to the initial comment related to this problem.